tasks/species_typing/task_tbprofiler_output_parsing.wdl

```diff
@@ -3,10 +3,12 @@ version 1.0
 task tbprofiler_output_parsing {
   input {
     File json
+    File gene_coverage
     String output_seq_method_type
     String operator
     String samplename
     Int min_depth = 10
+    Int coverage_threshold = 100
   }
   command <<<
     python3 <<CODE
```

Comment:
This is a lot of python code, that might be easier to maintain as a separate script